controllers/operatorconfigmap_controller.go

```diff
@@ -613,6 +623,25 @@ func (c *OperatorConfigMapReconciler) getDeployCSIConfig() (bool, error) {
 	return deployCSI, nil
 }
 
+func (c *OperatorConfigMapReconciler) checkForRemoteClient() (bool,error) {
+	clusterVersion := &configv1.ClusterVersion{}
+	clusterVersion.Name = clusterVersionName
+	if err := c.get(clusterVersion); err != nil {
+		c.log.Error(err, "failed to get the clusterVersion version of the OCP cluster")
+		return false, err
+	}
+	storageClients := &v1alpha1.StorageClientList{}
+	if err := c.list(storageClients); err != nil {
+		return false, fmt.Errorf("failed to list storageclients: %v", err)
+	}
+	for idx := range storageClients.Items {
```

Comment:
You should use the util.Find() to search through the loop.
https://github.com/red-hat-storage/ocs-client-operator/blob/ca0c95c9c552a7d0e1c60e592f036d6dad05546e/pkg/utils/slices.go#L20

Reply:
not sure what is the added value for that is